package eu.ehr4cr.workbench.local.exception.security;

public class InvitedUserExistsException extends RuntimeException {
}
